packages/notebook/src/widget.ts

```diff
@@ -1317,10 +1335,23 @@ class Notebook extends StaticNotebook {
         toMove.push(widget);
       }
     });
-
-    // Create the drag image.
-    let dragImage = Private.createDragImage(selected.length);
-
+    let activeCell = this.activeCell;
+    let dragImage: HTMLElement = null;
+    if (activeCell.model.type === 'code') {
+      let executionCount = (activeCell.model as ICodeCellModel).executionCount;
+      let countString: string;
+      if (executionCount) {
+        countString = executionCount.toString();
+      } else {
+        countString = '';
+      }
+      // Create the drag image.
+      dragImage = Private.createDragImage(selected.length, countString, activeCell.model.value.text.split('\n')[0].slice(0,26));
```

Comment:
I would move the `let countString` outside the `if (activeCell.model.type === 'code')` test and then set the `countString` for non-code cells to the empty string. Then you can have a single call to `Private.createDragImage` outside that outer if test.